src/table.scss

```diff
@@ -229,18 +232,18 @@ $block: #{$fd-namespace}-table;
         background-color: var(--sapList_Hover_Background);
       }
 
-      &:last-of-type {
+      &-last {
         border-right: $fd-table-fixed-cell-border;
+
+        @include fd-rtl() {
+          border-right: $fd-table-border;
+          border-left: $fd-table-fixed-cell-border;
+        }
```

Comment:
The exact same code is repeated twice. Can you move it to a placeholder? Thank you